docs/getting-started/index.md

```diff
@@ -0,0 +1,19 @@
+---
+layout: default
+title: Getting-started
+nav_order: 3
```

Comment:
should have a permalink here too, similar to components?

`permalink: /getting-started`